src/javascript/app/base/binary_loader.js

```diff
@@ -142,28 +142,27 @@ const BinaryLoader = (() => {
     };
 
     const handleNotAuthenticated = () => {
-        const content = container.querySelector('#content .container');
+        const content = container.querySelector('#content');
         if (!content) {
             return;
         }
-
-        const outer_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] });
-        const rowDiv = (element) => {
-            const row_element = createElement('div', { class: 'gr-padding-10' });
-            row_element.appendChild(element);
-            return row_element;
-        };
-        const inner_container = createElement('div', { class: 'center-text' });
-        const error_msg = createElement('div', { class: 'center-text notice-msg', text: error_messages.not_authenticated() });
-        const logout_cta = createElement('button');
-        const logout_span = createElement('span', { text: localize ('Sign out') });
-
-        logout_cta.addEventListener('click', () => { Client.doLogout({ logout: 1 }); });
-        logout_cta.appendChild(logout_span);
-        inner_container.appendChild(rowDiv(error_msg));
-        inner_container.appendChild(rowDiv(logout_cta));
-        outer_container.append(inner_container);
-        content.html(outer_container);
+        content.classList.add('container');
+        content.html(
+            `<div id="page-info">${ getElementById('page_info').outerHTML }</div>
+            <div class="logged_out_title_container">${ content.getElementsByTagName('h1')[0].outerHTML }</div>
+            <div class="center-text">
+                <div class="gr-padding-10">
+                    <div class="center-text notice-msg">${ error_messages.not_authenticated() }</div>
+                </div>
+                <div class="gr-padding-10">
+                    <button id="sign-out-button">
+                        <span>${ localize('Sign out') }</span>
+                    </button>
+                </div>
+            </div>`
```

Comment:
we don't generally set html like this, we create the divs. feel free to instead add the element as a layout component in pages that need it and make it visible when required instead.